apps/web/src/app/_components/search/company-search-bar.tsx

```diff
@@ -0,0 +1,182 @@
+              <FormControl>
+                <Select
+                  onValueChange={(value) => {
+                    setSelectedIndustry(value);
+                    const finalValue = value === "INDUSTRY" ? undefined : value;
+                    setValue(field.name, finalValue);
+                    void handleSubmit(onSubmit)();
+                  }}
+                  value={selectedIndustry}
+                >
+                  <SelectTrigger className="h-12 w-[340px] rounded-none border-2 border-l-0 border-t-0 border-[#9A9A9A] text-lg placeholder:opacity-50 focus:ring-0 active:ring-0 lg:rounded-md lg:border-2">
+                    <SelectValue placeholder="Industry" />
+                  </SelectTrigger>
+                  <SelectContent>
+                    <SelectGroup>
+                      <SelectItem className="font-bold" value="INDUSTRY">
+                        Industry
+                      </SelectItem>
+                      <SelectSeparator />
+                      {Object.entries(Industry).map(([key, value]) => (
+                        <SelectItem key={value} value={value}>
+                          {key.charAt(0) + key.slice(1).toLowerCase()}
+                        </SelectItem>
+                      ))}
+                    </SelectGroup>
+                  </SelectContent>
+                </Select>
+              </FormControl>
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="searchLocation"
+          render={({ field }) => (
+            <FormItem className="col-span-5 lg:col-span-2">
+              <FormControl>
+                <ComboBox
+                  {...field}
+                  variant="filtering"
+                  defaultLabel={locationLabel || "Location"}
+                  searchPlaceholder="Type to begin..."
+                  searchEmpty="No location found."
+                  valuesAndLabels={locationValuesAndLabels}
+                  currLabel={locationLabel}
+                  onChange={(value) => {
+                    setSearchTerm(value);
+                  }}
+                  onSelect={(currentValue) => {
+                    setLocationLabel(currentValue);
+                    const selectedLoc = locationsToUpdate.data?.find(
+                      (loc) =>
+                        `${loc.city}${loc.state ? `, ${loc.state}` : ""}${loc.country ? `, ${loc.country}` : ""}` ===
+                        currentValue,
+                    );
+                    const finalValue =
+                      currentValue === "LOCATION" ? undefined : selectedLoc?.id;
+                    setValue(field.name, finalValue);
+                    void handleSubmit(onSubmit)();
+                  }}
+                />
+              </FormControl>
+            </FormItem>
+          )}
+        />
+        <Button
+          className="bg-white hover:bg-white hover:text-[#9A9A9A] border-white text-black"
```

Comment:
Here we can make the text color here the same as the border color you used above to match the figma